.github/actions/setup-postgres/bin/pg_dump

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
this is bin file is not getting called anywhere?

Reply:
It's used for local dev, I didnt script anything else on top of it, instead I added the Usage + Installation section of this script's header description